[2024-09-25T23:38:31.085+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-25T23:38:31.115+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: spark_job_dag.submit_spark_job manual__2024-09-25T23:38:29.730842+00:00 [queued]>
[2024-09-25T23:38:31.129+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: spark_job_dag.submit_spark_job manual__2024-09-25T23:38:29.730842+00:00 [queued]>
[2024-09-25T23:38:31.129+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-09-25T23:38:31.146+0000] {taskinstance.py:2888} INFO - Executing <Task(SparkSubmitOperator): submit_spark_job> on 2024-09-25 23:38:29.730842+00:00
[2024-09-25T23:38:31.155+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'spark_job_dag', 'submit_spark_job', 'manual__2024-09-25T23:38:29.730842+00:00', '--job-id', '121', '--raw', '--subdir', 'DAGS_FOLDER/jobsSpark.py', '--cfg-path', '/tmp/tmpxcp48zrd']
[2024-09-25T23:38:31.152+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3579) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-25T23:38:31.156+0000] {standard_task_runner.py:105} INFO - Job 121: Subtask submit_spark_job
[2024-09-25T23:38:31.156+0000] {standard_task_runner.py:72} INFO - Started process 3582 to run task
[2024-09-25T23:38:31.227+0000] {task_command.py:467} INFO - Running <TaskInstance: spark_job_dag.submit_spark_job manual__2024-09-25T23:38:29.730842+00:00 [running]> on host ede4b72e87e0
[2024-09-25T23:38:31.369+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='spark_job_dag' AIRFLOW_CTX_TASK_ID='submit_spark_job' AIRFLOW_CTX_EXECUTION_DATE='2024-09-25T23:38:29.730842+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-25T23:38:29.730842+00:00'
[2024-09-25T23:38:31.372+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-25T23:38:31.451+0000] {base.py:84} INFO - Retrieving connection 'spark_default'
[2024-09-25T23:38:31.598+0000] {spark_submit.py:403} INFO - Spark-Submit cmd: spark-submit --master spark://spark:7077 --num-executors 2 --executor-memory 1G --driver-memory 512M --name arrow-spark --deploy-mode client /opt/bitnami/spark/dags/my_spark_app.py
[2024-09-25T23:38:35.546+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:35 INFO SparkContext: Running Spark version 3.5.2
[2024-09-25T23:38:35.551+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:35 INFO SparkContext: OS info Linux, 5.15.153.1-microsoft-standard-WSL2, amd64
[2024-09-25T23:38:35.552+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:35 INFO SparkContext: Java version 17.0.12
[2024-09-25T23:38:35.660+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:35 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2024-09-25T23:38:35.791+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:35 INFO ResourceUtils: ==============================================================
[2024-09-25T23:38:35.792+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:35 INFO ResourceUtils: No custom resources configured for spark.driver.
[2024-09-25T23:38:35.792+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:35 INFO ResourceUtils: ==============================================================
[2024-09-25T23:38:35.793+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:35 INFO SparkContext: Submitted application: Airflow Spark Example
[2024-09-25T23:38:35.830+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:35 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2024-09-25T23:38:35.859+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:35 INFO ResourceProfile: Limiting resource is cpu
[2024-09-25T23:38:35.861+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:35 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2024-09-25T23:38:35.962+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:35 INFO SecurityManager: Changing view acls to: ***
[2024-09-25T23:38:35.963+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:35 INFO SecurityManager: Changing modify acls to: ***
[2024-09-25T23:38:35.964+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:35 INFO SecurityManager: Changing view acls groups to:
[2024-09-25T23:38:35.965+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:35 INFO SecurityManager: Changing modify acls groups to:
[2024-09-25T23:38:35.967+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:35 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users with view permissions: ***; groups with view permissions: EMPTY; users with modify permissions: ***; groups with modify permissions: EMPTY
[2024-09-25T23:38:36.711+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:36 INFO Utils: Successfully started service 'sparkDriver' on port 39797.
[2024-09-25T23:38:36.773+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:36 INFO SparkEnv: Registering MapOutputTracker
[2024-09-25T23:38:36.838+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:36 INFO SparkEnv: Registering BlockManagerMaster
[2024-09-25T23:38:36.854+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:36 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2024-09-25T23:38:36.855+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:36 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2024-09-25T23:38:36.874+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:36 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2024-09-25T23:38:36.886+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:36 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-efff1a99-85ed-468e-8cbf-39f77efed969
[2024-09-25T23:38:36.906+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:36 INFO MemoryStore: MemoryStore started with capacity 127.2 MiB
[2024-09-25T23:38:36.931+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:36 INFO SparkEnv: Registering OutputCommitCoordinator
[2024-09-25T23:38:37.172+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:37 INFO JettyUtils: Start Jetty 0.0.0.0:4040 for SparkUI
[2024-09-25T23:38:37.447+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:37 WARN Utils: Service 'SparkUI' could not bind on port 4040. Attempting port 4041.
[2024-09-25T23:38:37.463+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:37 INFO Utils: Successfully started service 'SparkUI' on port 4041.
[2024-09-25T23:38:37.821+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:37 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2024-09-25T23:38:37.882+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:37 INFO TransportClientFactory: Successfully created connection to spark/172.18.0.5:7077 after 38 ms (0 ms spent in bootstraps)
[2024-09-25T23:38:38.021+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:38 INFO StandaloneSchedulerBackend: Connected to Spark cluster with app ID app-20240925233837-0014
[2024-09-25T23:38:38.022+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:38 INFO StandaloneAppClient$ClientEndpoint: Executor added: app-20240925233837-0014/0 on worker-20240925215139-172.18.0.4-45621 (172.18.0.4:45621) with 6 core(s)
[2024-09-25T23:38:38.023+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:38 INFO StandaloneSchedulerBackend: Granted executor ID app-20240925233837-0014/0 on hostPort 172.18.0.4:45621 with 6 core(s), 1024.0 MiB RAM
[2024-09-25T23:38:38.027+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:38 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 36427.
[2024-09-25T23:38:38.027+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:38 INFO NettyBlockTransferService: Server created on ede4b72e87e0:36427
[2024-09-25T23:38:38.027+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:38 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2024-09-25T23:38:38.034+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:38 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, ede4b72e87e0, 36427, None)
[2024-09-25T23:38:38.036+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:38 INFO BlockManagerMasterEndpoint: Registering block manager ede4b72e87e0:36427 with 127.2 MiB RAM, BlockManagerId(driver, ede4b72e87e0, 36427, None)
[2024-09-25T23:38:38.039+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:38 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, ede4b72e87e0, 36427, None)
[2024-09-25T23:38:38.048+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:38 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, ede4b72e87e0, 36427, None)
[2024-09-25T23:38:38.081+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:38 INFO StandaloneAppClient$ClientEndpoint: Executor updated: app-20240925233837-0014/0 is now RUNNING
[2024-09-25T23:38:38.347+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:38 INFO StandaloneSchedulerBackend: SchedulerBackend is ready for scheduling beginning after reached minRegisteredResourcesRatio: 0.0
[2024-09-25T23:38:38.646+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:38 INFO SharedState: Setting hive.metastore.warehouse.dir ('null') to the value of spark.sql.warehouse.dir.
[2024-09-25T23:38:38.653+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:38 INFO SharedState: Warehouse path is 'file:/opt/***/spark-warehouse'.
[2024-09-25T23:38:40.161+0000] {spark_submit.py:579} INFO - Traceback (most recent call last):
[2024-09-25T23:38:40.171+0000] {spark_submit.py:579} INFO - File "/opt/bitnami/spark/dags/my_spark_app.py", line 104, in <module>
[2024-09-25T23:38:40.181+0000] {spark_submit.py:579} INFO - df_placa_mae, df_filtred_memoria, df_processadores, df_placa_video =transformation_data('opt/***/datalake/rawzone/kabum=2024-09-23/data.json')
[2024-09-25T23:38:40.185+0000] {spark_submit.py:579} INFO - ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2024-09-25T23:38:40.192+0000] {spark_submit.py:579} INFO - File "/opt/bitnami/spark/dags/my_spark_app.py", line 22, in transformation_data
[2024-09-25T23:38:40.198+0000] {spark_submit.py:579} INFO - df=spark.read.json(path)
[2024-09-25T23:38:40.199+0000] {spark_submit.py:579} INFO - ^^^^^^^^^^^^^^^^^^^^^
[2024-09-25T23:38:40.200+0000] {spark_submit.py:579} INFO - File "/home/***/.local/lib/python3.12/site-packages/pyspark/python/lib/pyspark.zip/pyspark/sql/readwriter.py", line 425, in json
[2024-09-25T23:38:40.200+0000] {spark_submit.py:579} INFO - File "/home/***/.local/lib/python3.12/site-packages/pyspark/python/lib/py4j-0.10.9.7-src.zip/py4j/java_gateway.py", line 1322, in __call__
[2024-09-25T23:38:40.201+0000] {spark_submit.py:579} INFO - File "/home/***/.local/lib/python3.12/site-packages/pyspark/python/lib/pyspark.zip/pyspark/errors/exceptions/captured.py", line 185, in deco
[2024-09-25T23:38:40.201+0000] {spark_submit.py:579} INFO - pyspark.errors.exceptions.captured.AnalysisException: [PATH_NOT_FOUND] Path does not exist: file:/opt/***/opt/***/datalake/rawzone/kabum=2024-09-23/data.json.
[2024-09-25T23:38:40.263+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:40 INFO SparkContext: Invoking stop() from shutdown hook
[2024-09-25T23:38:40.264+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:40 INFO SparkContext: SparkContext is stopping with exitCode 0.
[2024-09-25T23:38:40.279+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:40 INFO SparkUI: Stopped Spark web UI at http://ede4b72e87e0:4041
[2024-09-25T23:38:40.283+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:40 INFO StandaloneSchedulerBackend: Shutting down all executors
[2024-09-25T23:38:40.286+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:40 INFO StandaloneSchedulerBackend$StandaloneDriverEndpoint: Asking each executor to shut down
[2024-09-25T23:38:40.307+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:40 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2024-09-25T23:38:40.331+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:40 INFO MemoryStore: MemoryStore cleared
[2024-09-25T23:38:40.331+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:40 INFO BlockManager: BlockManager stopped
[2024-09-25T23:38:40.338+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:40 INFO BlockManagerMaster: BlockManagerMaster stopped
[2024-09-25T23:38:40.340+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:40 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2024-09-25T23:38:40.351+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:40 INFO SparkContext: Successfully stopped SparkContext
[2024-09-25T23:38:40.352+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:40 INFO ShutdownHookManager: Shutdown hook called
[2024-09-25T23:38:40.352+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:40 INFO ShutdownHookManager: Deleting directory /tmp/spark-3ceb495e-ffa6-4927-a0fd-452311873645/pyspark-16a7d6b4-539c-444a-aa08-4ca13bfcde3f
[2024-09-25T23:38:40.356+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:40 INFO ShutdownHookManager: Deleting directory /tmp/spark-cfc5ca00-f47d-47ff-bd13-3cd0bca0d84a
[2024-09-25T23:38:40.362+0000] {spark_submit.py:579} INFO - 24/09/25 23:38:40 INFO ShutdownHookManager: Deleting directory /tmp/spark-3ceb495e-ffa6-4927-a0fd-452311873645
[2024-09-25T23:38:40.397+0000] {taskinstance.py:3310} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 174, in execute
    self._hook.submit(self.application)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 505, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --num-executors 2 --executor-memory 1G --driver-memory 512M --name arrow-spark --deploy-mode client /opt/bitnami/spark/dags/my_spark_app.py. Error code is: 1.
[2024-09-25T23:38:40.405+0000] {taskinstance.py:1225} INFO - Marking task as UP_FOR_RETRY. dag_id=spark_job_dag, task_id=submit_spark_job, run_id=manual__2024-09-25T23:38:29.730842+00:00, execution_date=20240925T233829, start_date=20240925T233831, end_date=20240925T233840
[2024-09-25T23:38:40.423+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-25T23:38:40.423+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 121 for task submit_spark_job (Cannot execute: spark-submit --master spark://spark:7077 --num-executors 2 --executor-memory 1G --driver-memory 512M --name arrow-spark --deploy-mode client /opt/bitnami/spark/dags/my_spark_app.py. Error code is: 1.; 3582)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/cli.py", line 115, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3004, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3158, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3182, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 174, in execute
    self._hook.submit(self.application)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 505, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --num-executors 2 --executor-memory 1G --driver-memory 512M --name arrow-spark --deploy-mode client /opt/bitnami/spark/dags/my_spark_app.py. Error code is: 1.
[2024-09-25T23:38:40.464+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2024-09-25T23:38:40.481+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-25T23:38:40.486+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
